spec/lob/v1/bank_account_spec.rb

```diff
@@ -49,9 +33,7 @@
     it "should create a bank account with address params" do
```

Comment:
you can probably just remove this test since it's just like the one above and rename that one to just `create a bank account`